package.json

```diff
@@ -33,6 +33,8 @@
     "sentry:publish": "node ./development/sentry-publish.js",
     "lint": "eslint . --ext js,json",
     "lint:fix": "eslint . --ext js,json --fix",
+    "lint:diffed": "eslint $(git diff --name-only) $(git diff --cached --name-only) --ext js,json",
```

Comment:
Unfortunately this strategy of using an unquoted command substitution is susceptible to word-splitting problems. Any files with spaces in the name will break, for example.

Perhaps the two commands could be combined, then piped to `xargs` and passed to `eslint`. Like this:

```
{ git ls-files --others --exclude-standard ; git diff-index --name-only --diff-filter=d HEAD ; } | xargs -d '\n' eslint
```